guvnor-structure/guvnor-structure-client/src/main/java/org/guvnor/structure/client/editors/repository/clone/CloneRepositoryView.java

```diff
@@ -0,0 +1,87 @@
+package org.guvnor.structure.client.editors.repository.clone;
+
+import com.github.gwtbootstrap.client.ui.constants.ControlGroupType;
+
+public interface CloneRepositoryView {
```

Comment:
You can extend ```org.uberfire.ext.widgets.common.client.common.HasBusyIndicator``` (which has the methods to show/hide a "<doing something>..." popup message to Users (instead of having ```showBusyPopupMessage()``` and ```closeBusyPopup()```).